tests/data/format/no_whitespace_stripper/module_docstring.py.out

```diff
@@ -0,0 +1,18 @@
+"""  A multi-line  .
```

Comment:
I wonder if we should rstrip the end of the line before adding the final dot. Thought @DanielNoord ?

Reply:
I was thinking the same thing while writing the test, but I figured at the end of the day the argument is called `--no-strip-whitespace` .... so I figured it was consistent in its behavior.

Maybe down the road we could change the argument such that there is the possibility of only removing trailing whitespace, instead of always removing both trailing and starting.

LMK what you think

Reply:
Yeah agreed.

I think if you're letting a tool put periods in your docstrings automatically you'll likely also let that tool remove whitespaces. So they probably won't be without the other very often. And in those cases where people do I think it is good to offer the flexibility.